CHANGELOG.md

```diff
@@ -175,6 +175,9 @@ Add Brazilian Portuguese translation (pt-BR.json) by [@pstwh](http://github.com/
 
 <img width="951" alt="image" src="https://user-images.githubusercontent.com/1778297/206615305-4c52031e-3f7d-4df2-8805-a79894206911.png">
 
+### Add Waveform Visual Support to Audio
```

Comment:
Add this to the latest release section please

Reply:
done.